src/components/Person/Person.jsx

```diff
@@ -1 +1,30 @@
 // export const Person = ({ person }) => ();
+
+export const Person = ({ person }) => {
+  const isMarried = person.sex === 'm' ? 'wife' : 'husband';
```

Comment:
we usually name boolean vars starting with `is`, so better to rename this var here